website/docs/docs/dbt-versions/2023-release-notes.md

```diff
@@ -0,0 +1,998 @@
+    - If you edit the `dependencies.yml` file and the number of packages remains the same, run `dbt deps`. (Note that this is a known bug dbt Labs will fix in the future.)
+
+  </expandable>
+
+- <expandable alt_header='Native retry support'>
+
+    Previously in dbt Cloud, you could only rerun an errored job from start but now you can also rerun it from its point of failure. 
+
+    You can view which job failed to complete successully, which command failed in the run step, and choose how to rerun it. To learn more, refer to [Retry jobs](/docs/deploy/retry-jobs).
+
+    <Lightbox src="/img/docs/deploy/native-retry.gif" width="70%" title="Example of the Rerun options in dbt Cloud"/>
+
+  </expandable>
+
+- <expandable alt_header='Product docs updates'>
+
+    Hello from the dbt Docs team: @mirnawong1, @matthewshaver, @nghi-ly, and @runleonarun! First, we’d like to thank the 15 new community contributors to docs.getdbt.com. We merged [107 PRs](https://github.com/dbt-labs/docs.getdbt.com/pulls?q=is%3Apr+merged%3A2023-09-01..2023-09-31) in September.
+
+    Here's what's new to [docs.getdbt.com](http://docs.getdbt.com/):
+
+    * Migrated docs.getdbt.com from Netlify to Vercel.
+
+    ## ☁ Cloud projects
+    - Continuous integration jobs are now generally available and no longer in beta!
+    - Added [Postgres PrivateLink set up page](/docs/cloud/secure/postgres-privatelink)
+    - Published beta docs for [dbt Explorer](/docs/collaborate/explore-projects).
+    - Added a new Semantic Layer [GraphQL API doc](/docs/dbt-cloud-apis/sl-graphql) and updated the [integration docs](/docs/use-dbt-semantic-layer/avail-sl-integrations) to include Hex. Responded to dbt community feedback and clarified Metricflow use cases for dbt Core and dbt Cloud.
+    - Added an [FAQ](/faqs/Git/git-migration) describing how to migrate from one git provider to another in dbt Cloud.
+    - Clarified an example and added a [troubleshooting section](/docs/cloud/connect-data-platform/connect-snowflake#troubleshooting) to Snowflake connection docs to address common errors and provide solutions.
+
+    ## 🎯 Core projects
+
+    - Deprecated dbt Core v1.0 and v1.1 from the docs.
+    - Added configuration instructions for the [AWS Glue](/docs/core/connect-data-platform/glue-setup) community plugin.
+    - Revised the dbt Core quickstart, making it easier to follow. Divided this guide into steps that align with the [other guides](/guides/manual-install?step=1).
+
+    ## New 📚 Guides, ✏️ blog posts, and FAQs
+
+    Added a [style guide template](/best-practices/how-we-style/6-how-we-style-conclusion#style-guide-template) that you can copy & paste to make sure you adhere to best practices when styling dbt projects!
+
+    ## Upcoming changes
+
+    Stay tuned for a flurry of releases in October and a filterable guides section that will make guides easier to find!
+
+  </expandable>
+
+- <expandable alt_header='Semantic layer GA'>
+
```

Comment:
fyi this is rendering weird -- might be the spacing for the :::

![Screenshot 2024-05-15 at 12 24 44](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/b7dc9209-f354-4c28-be30-000e3b12136d)

Reply:
Looks like the callout functionality doesn't work under the expandables. Removing the formatting